### DISCUSSION TITLE
cpu 'all' line

### DISCUSSION THREAD

**Discussion by retroflexivity** [ClementTsang/bottom#1610] [Q&A] [answered]
In the cpu legend, there is a line that only states 'all'. It shows no usage, it is not interactive, and it takes a whole line. Hence I'm wondering what is its _raison d'etre_ and whether it is possible to hide it.

![image](https://github.com/user-attachments/assets/47367795-ebe2-4278-9c3a-06918baba82f)

**Comment by ClementTsang** [comment, marked as answer]
It's so a user can select whether they want to show all cores at once in the graph. That's it.

You cannot hide it at the moment, but I can certainly look at doing that.

**Reply by retroflexivity** [reply]
oh, of course. thank you! I believe key bindings (like nums and e.g. 'a') could be a great alternative way to toggle between cores
